synchronization/block_tracker.go

```diff
@@ -15,9 +15,7 @@ type BlockTracker struct {
 	currentHeight uint64 // this is not primitives.BlockHeight so as to avoid unnecessary casts
 	latch         chan struct{}
 
-	// following fields are for tests only
-	enteredSelectSignalForTests chan int
-	selectIterationsForTests    int
+	fireOnWait func() // used by unit test
```

Comment:
this is a non-exported field; is this feature *only* used by the block tracker's own unit tests?